save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -128,20 +130,32 @@ class FixPlugin(
                 }
             }
 
-    private fun createTestFile(path: Path, generalConfig: GeneralConfig): Path {
+    private fun createTestFile(
+        path: Path,
+        generalConfig: GeneralConfig,
+        fixPluginConfig: FixPluginConfig): Path {
         val pathCopy: Path = constructPathForCopyOfTestFile(FixPlugin::class.simpleName!!, path)
         createTempDir(pathCopy.parent!!)
 
-        val expectedWarningPattern = generalConfig.expectedWarningsPattern
+        val defaultIgnoreLinesPatterns: MutableList<Regex> = mutableListOf()
+        generalConfig.expectedWarningsPattern?.let { defaultIgnoreLinesPatterns.add(it) }
+        generalConfig.runConfigPattern?.let { defaultIgnoreLinesPatterns.add(it) }
 
         fs.write(fs.createFile(pathCopy)) {
-            fs.readLines(path).forEach {
-                if (expectedWarningPattern == null || !generalConfig.expectedWarningsPattern!!.matches(it)) {
+            fs.readLines(path)
+                .filter { line ->
+                    fixPluginConfig.ignoreLines?.let {
+                        fixPluginConfig.ignoreLinesPatterns.none { pattern -> pattern.matches(line) }
+                    }
+                        ?: run {
+                            defaultIgnoreLinesPatterns.none {regex -> regex.matches(line) }
```

Comment:
diktat is not working? where are spaces after `{`

Reply:
gradlew.bat diktatFixAll

Reply:
Interesting. Seems that this case is ignored by diktat.